management/src/test/java/org/ehcache/management/providers/statistics/HitRateTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.management.providers.statistics;
+
+import static java.util.Arrays.asList;
+import static org.ehcache.config.builders.ResourcePoolsBuilder.newResourcePoolsBuilder;
+import static org.ehcache.config.units.MemoryUnit.MB;
+import static org.ehcache.config.units.EntryUnit.ENTRIES;
+import static org.hamcrest.CoreMatchers.is;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.ehcache.Cache;
+import org.ehcache.CacheManager;
+import org.ehcache.config.Builder;
+import org.ehcache.config.CacheConfiguration;
+import org.ehcache.config.EvictionAdvisor;
+import org.ehcache.config.ResourcePools;
+import org.ehcache.config.builders.CacheConfigurationBuilder;
+import org.ehcache.config.builders.CacheManagerBuilder;
+import org.ehcache.impl.config.persistence.DefaultPersistenceConfiguration;
+import org.ehcache.management.ManagementRegistryService;
+import org.ehcache.management.config.EhcacheStatisticsProviderConfiguration;
+import org.ehcache.management.registry.DefaultManagementRegistryConfiguration;
+import org.ehcache.management.registry.DefaultManagementRegistryService;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.terracotta.management.model.context.Context;
+
+@RunWith(Parameterized.class)
+public class HitRateTest {
+
+  @Rule
+  public final Timeout globalTimeout = Timeout.seconds(10);
+
+  @Rule
+  public final TemporaryFolder diskPath = new TemporaryFolder();
+
+  private static final EhcacheStatisticsProviderConfiguration EHCACHE_STATISTICS_PROVIDER_CONFIG = new EhcacheStatisticsProviderConfiguration(1,TimeUnit.MINUTES,100,1,TimeUnit.MILLISECONDS,10,TimeUnit.MINUTES);
```

Comment:
I will make the change. I changed it in the latency tests but I also planned to update the rest of the stats test in a separate task.